ui/app/components/app/provider-page-container/provider-page-container.component.js

```diff
@@ -18,8 +18,11 @@ export default class ProviderPageContainer extends PureComponent {
     t: PropTypes.func,
     metricsEvent: PropTypes.func,
   };
-
+  componentWillUnmount () {
+    window.onbeforeunload = null
```

Comment:
Generally it's better to use `window.addEventListener("beforeunload", function(event) { ... });` instead of `window.onbeforeunload` because it avoids overwriting other `onbeforeunload` handlers (e.g. the pattern I used in #7333)